packages/server/src/api/controllers/view/exporters.ts

```diff
@@ -1,7 +1,19 @@
 import { Row, TableSchema } from "@budibase/types"
 
-export function csv(headers: string[], rows: Row[]) {
-  let csv = headers.map(key => `"${key}"`).join(",")
+function getHeaders(
+  headers: string[],
+  customHeaders: { [key: string]: string }
+) {
+  return headers.map(header => `"${customHeaders[header] || header}"`)
```

Comment:
Is it strictly correct to wrap this in quotes manually? What if the header already contains quotes?

Reply:
The export data action has always been wrapping the values, including the headers, in double quotes. I imagine now that we have added space as a delimiter, the quotes will be necessary.

Reply:
Fair enough. It will break if users have a `"` in their headers, but if it's been there forever that's a separate bug we can address if/when it comes up.